.github/workflows/rtds_build_and_push_dev.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
           while ! docker exec rtdsagent-1 nc localhost 4343 ; do sleep 1; done;
           while ! docker exec rtdsagent-2 nc localhost 4343 ; do sleep 1; done;
           pytest --ignore tests/test_clean_install.py
+          pytest --ignore tests/test_user.py
```

Comment:
This must be on single line - this is supposed to run all tests except the one(s) listed in --ignore.

Reply:
You also run the same tests twice - the second iteration fails, as the channels etc. already exist.